src/apps/vpn/connectionbenchmark/connectionbenchmark.cpp

```diff
@@ -218,11 +222,14 @@ void ConnectionBenchmark::handleStabilityChange() {
   if (m_state == StateInitial || m_state == StateReady) {
     return;
   }
+  logger.debug() << "Handle stability change";
```

Comment:
Move the log statement above the if/return

Reply:
This would be a change for this function, as it previously behaved similarly (before I broke it into two functions) - and breaks the pattern shared between it and `handleControllerState`. The function called a lot, and so I think I like keeping it as "only log if there is an update". Can you explain why you think we should change the logging here?

Reply:
My justification for moving the log statement to the first line would be that if the `if` statement returns then we wouldn't know that we entered that function. With that being said, I'm unaware of how much it is being called so if you feel like it would clutter up the logging, I'm totally fine with leaving it as is.

Reply:
Thanks - I'm inclined to leave it as it was originally.